source/coll_db.f90

```diff
@@ -38,6 +41,7 @@ module coll_db
   ! Family Arrays
   character(len=:), allocatable, public, save :: cdb_famName(:)     ! Family name
   real(kind=fPrec), allocatable, public, save :: cdb_famNSig(:)     ! Family sigma
+  real(kind=fPrec), allocatable, public, save :: cdb_famNSigOrig(:) ! Family sigma
```

Comment:
maybe add a comment line saying that the original nSigma is useful for DYNKing the collimator settings - or do I wrongly remember?

Reply:
I forgot now. Will handle all of that when I finish the last PR in this chain where the DYNK stuff is applied. I think I need to make some changes to this (internally) anyway.